modules/flowable-cmmn-api/src/main/java/org/flowable/cmmn/api/history/HistoricPlanItemInstanceQuery.java

```diff
@@ -12,12 +12,12 @@
  */
 package org.flowable.cmmn.api.history;
 
+import org.flowable.common.engine.api.query.Query;
```

Comment:
Please note that project standards specify that imports for java classes are before those of `com` and `org` as was done originally in the file.  Thanks.